### DISCUSSION TITLE
Will there be a stable release soon?

### DISCUSSION THREAD

**Discussion by NotepadPlusUser** [Open-Shell/Open-Shell-Menu#1302] [General]
I have been following the successive pre-release versions on Github with great interest, but not being a techie, I am very nervous about installing a pre-release version.  My new desktop is Windows 11, and I have already wasted a lot of time battling its bugs and annoyances.

I installed Open-Shell with no problems at all --- wonderful --- although I am only using the Start Menu replacement part of the software.

Do you know yet when a stable, Windows-11-compatible, version will be released?  

 . . . and thank you so much for this software.  For years now I have appreciated very much not having to cope with all the nonsense and confusion on the M$ Start Menu.  So much work by you and other developers to set all sorts of things in Windows 11 back to the excellent situation that they were in only a few short years ago!

**Comment by ge0rdi** [comment]
I was thinking about doing official release (most likely from 4.4.189) because there were many Windows 11 improvements done in start menu.
So if there won't be any issues reported (say in a week), I'll release it officially.

>  I am very nervous about installing a pre-release version

In such case you should simply wait for official release.
Pre-releases are mostly for people that can handle eventual troubles (we are always trying to avoid any) and that are willing to report eventual regressions.

Though, please understand that even official release doesn't guarantee there will be no issues.

> Do you know yet when a stable, Windows-11-compatible, version will be released?

Just note that while there were many Windows 11 improvements done in current pre-release versions, there are still many things left (especially for `Classic Explorer`).

**Reply by NotepadPlusUser** [reply]
Thanks very much for your rep[y, and for all your hard work.  I understand that we can't have a definitive release yet, because Windows 11 is still going to be a work in progress for a long time.  But a (reasonably) bug-free update to deal with the present state of Windows 11 would be very welcome, even if a future Windows 11 update breaks something,

Right on cue, I ran into another Windows 11 bug?/feature? yesterday (I have only had it for a couple of weeks).  I turned on the Size column in File Explorer, and found that the size of directories is not listed!  DuckDuckGoing established that there was not even a decent hack to turn this on in the Size column.  I rushed back to Directory Opus, wondering why M$ was destroying itself.
